src/_assets/css/main.scss

```diff
@@ -394,10 +395,12 @@ img {
 // #footernav {}
 
 #sidenav {
-  position: relative;
+  position: fixed;
```

Comment:
I always thought `position: fixed` is dangerous and/or unsupported. But maybe that's an outdated notion. 

Please make sure, for example, that one the sidenav is `position: fixed`, you can actually scroll the contents of the sidebar when the window is too short.

Reply:
Flutter uses a similar `position:fixed`-style sidebar, that's the reason I've moved this. I've checked scrolling (with Chrome on Mac).